### ISSUE TITLE
Pre-work Checklist: Developer: Angelina Hayrapetyan

### ISSUE THREAD

**Issue by angieyan** [hackforla/website#3813]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] #4001
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@angieyan Are you joining us? If so, please work on the checkboxes above. If not, please let us know so that we can close out this issue.

**Comment by angieyan** [comment]
Provide Update
1. Progress - In progress
2. Blockers - Waiting to be added to the Drive
3. Availability - Mon-Thurs 9am-10am
4. ETA 02/17/23

How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
Total 7 hours

**Comment by angieyan** [comment]
1. Progress - Completed
2. Blockers - None
3. Availability
4. ETA - 02/17/23

**Comment by angieyan** [comment]
Progress - Completed
Blockers - None
Availability - Mar 3 10am-11am
ETA - 02/17/23

**Comment by angieyan** [comment]
1. I have completed all. Waiting for approval
2. No blockers
3. N/A
4. It is already done
5. No visual changes

**Comment by ExperimentsInHonesty** [comment]
@angieyan 

There are now

1 [back end Medium size issues ](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22+-label%3A%22role%3A+dev+leads%22#column-7198257)

AND 

2 [front end Medium size issues](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257)

Available.  

That may change by the time you are ready to work on them, and we are always working on new ones.  

Please pick an issue from this link when you are ready to work on it.  If there are no issues at the link, when you are ready, write us back here, and I will keep an eye out for you again.  Please don't pick up an issue unless you will be working on it in 24 hours or less.

**Comment by angieyan** [comment]
@ExperimentsInHonesty 
Thank you for the issues.
I took the first one.
will be working on it today and on Friday.

**Comment by jdingeman** [comment]
Hi @angieyan - I know you are currently active, please provide us with an update on your overall progress. Thanks!

**Comment by angieyan** [comment]
@jdingeman I thought I am done with this issue. Just changed "in progress" to "done". Please tell me what else I should do.

**Comment by roslynwythe** [comment]
Hi @angieyan.  Please create an update as described in the above comment.  If you are not currently working on other issues, then we would like to see updates at least every two weeks, to inform us of your progress and any problems you might be experiencing, and to let us know that you are still interested in participating in Hack for LA.  

If we do not hear from you by 7/10/2023, we will assume that you are not interested in participating in Hack for LA at this time, and we will close this issue, but you can reopen it at a later date, and move it into the "In Progress" column.  Thank you.

**Comment by HackforLABot** [comment]
## Activity Log: angieyan
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- angieyan opened: https://github.com/hackforla/website/issues/3813 at 01/12/2023, 07:29 PM PST
- angieyan assigned: https://github.com/hackforla/website/issues/3851 at 02/10/2023, 12:28 PM PST
- angieyan commented on: https://github.com/hackforla/website/issues/3851#issuecomment-1426307601 at 02/10/2023, 12:29 PM PST
- angieyan opened: https://github.com/hackforla/website/pull/3965 at 02/12/2023, 06:41 PM PST
- angieyan assigned: https://github.com/hackforla/website/issues/3813#issuecomment-1427330989 at 02/12/2023, 09:23 PM PST
- angieyan commented on: https://github.com/hackforla/website/issues/3813#issuecomment-1427416118 at 02/12/2023, 10:25 PM PST
- angieyan closed by PR 4107: https://github.com/hackforla/website/issues/3925#event-8505706819 at 02/13/2023, 07:21 AM PST
- angieyan reopened: https://github.com/hackforla/website/issues/3925#event-8505706819 at 02/13/2023, 07:21 AM PST
- angieyan commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428599003 at 02/13/2023, 12:03 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428600390 at 02/13/2023, 12:04 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428751449 at 02/13/2023, 02:02 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428926025 at 02/13/2023, 04:43 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428969481 at 02/13/2023, 05:29 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428976589 at 02/13/2023, 05:36 PM PST
- angieyan PR was merged: https://github.com/hackforla/website/pull/3965#event-8516921123 at 02/14/2023, 07:41 AM PST
- angieyan issue closed by PR 3965: https://github.com/hackforla/website/issues/3851#event-8516921860 at 02/14/2023, 07:41 AM PST
- angieyan commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1430345910 at 02/14/2023, 12:40 PM PST
- angieyan commented on: https://github.com/hackforla/website/issues/3813#issuecomment-1435477692 at 02/17/2023, 08:00 PM PST
- angieyan assigned: https://github.com/hackforla/website/issues/3952#issuecomment-1421840677 at 02/24/2023, 08:35 PM PST
- angieyan commented on: https://github.com/hackforla/website/issues/3952#issuecomment-1444989561 at 02/24/2023, 08:37 PM PST
- angieyan opened: https://github.com/hackforla/website/pull/4038 at 02/24/2023, 09:15 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/4038#issuecomment-1445260004 at 02/25/2023, 07:35 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/4038#issuecomment-1447064034 at 02/27/2023, 12:47 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/4038#issuecomment-1449351605 at 02/28/2023, 09:02 PM PST
- angieyan PR was merged: https://github.com/hackforla/website/pull/4038#event-8634771442 at 02/28/2023, 11:17 PM PST
- angieyan issue closed by PR 4038: https://github.com/hackforla/website/issues/3952#event-8634771589 at 02/28/2023, 11:17 PM PST
- angieyan commented on: https://github.com/hackforla/website/issues/3813#issuecomment-1453874027 at 03/03/2023, 09:38 AM PST
- angieyan assigned: https://github.com/hackforla/website/issues/3720#issuecomment-1436073915 at 03/05/2023, 07:13 AM PST
- angieyan commented on: https://github.com/hackforla/website/issues/3720#issuecomment-1455119430 at 03/05/2023, 07:15 AM PST
- angieyan opened: https://github.com/hackforla/website/pull/4115 at 03/05/2023, 12:42 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/4115#issuecomment-1459117385 at 03/07/2023, 05:18 PM PST
- angieyan opened: https://github.com/hackforla/website/pull/4131 at 03/07/2023, 09:51 PM PST
- angieyan PR was closed: https://github.com/hackforla/website/pull/4115#event-8693331373 at 03/07/2023, 10:09 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/4131#issuecomment-1462763207 at 03/09/2023, 12:41 PM PST
- angieyan commented on: https://github.com/hackforla/website/pull/4131#issuecomment-1465098537 at 03/11/2023, 09:45 PM PST
- angieyan PR was merged: https://github.com/hackforla/website/pull/4131#event-8749268906 at 03/14/2023, 04:03 PM PDT
- angieyan issue closed by PR 4131: https://github.com/hackforla/website/issues/3720#event-8749269118 at 03/14/2023, 04:03 PM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/3813#issuecomment-1483661404 at 03/24/2023, 06:26 PM PDT
- angieyan assigned: https://github.com/hackforla/website/issues/2652#issuecomment-1475355262 at 03/27/2023, 11:09 AM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/3813#issuecomment-1485619942 at 03/27/2023, 11:11 AM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/2652#issuecomment-1485626191 at 03/27/2023, 11:14 AM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/2652#issuecomment-1495027559 at 04/03/2023, 02:47 PM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/2652#issuecomment-1504349480 at 04/11/2023, 05:49 PM PDT
- angieyan opened: https://github.com/hackforla/website/issues/4502 at 04/13/2023, 10:28 AM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/3813#issuecomment-1512333764 at 04/17/2023, 07:06 PM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4524#issuecomment-1513712026 at 04/18/2023, 12:49 PM PDT
- angieyan submitted review: https://github.com/hackforla/website/pull/4524#pullrequestreview-1390829485 at 04/18/2023, 12:49 PM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4497#issuecomment-1513761149 at 04/18/2023, 01:32 PM PDT
- angieyan submitted review: https://github.com/hackforla/website/pull/4497#pullrequestreview-1390905473 at 04/18/2023, 01:42 PM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4511#issuecomment-1513775033 at 04/18/2023, 01:46 PM PDT
- angieyan submitted review: https://github.com/hackforla/website/pull/4511#pullrequestreview-1390913537 at 04/18/2023, 01:48 PM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4558#issuecomment-1519165947 at 04/23/2023, 01:55 PM PDT
- angieyan submitted review: https://github.com/hackforla/website/pull/4558#pullrequestreview-1396988571 at 04/23/2023, 01:58 PM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4550#issuecomment-1519167532 at 04/23/2023, 02:02 PM PDT
- angieyan issue closed as not planned: https://github.com/hackforla/website/issues/2652#event-9087278250 at 04/24/2023, 07:14 PM PDT
- angieyan assigned: https://github.com/hackforla/website/issues/4590 at 05/01/2023, 07:56 PM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/4590#issuecomment-1530799729 at 05/01/2023, 07:58 PM PDT
- angieyan commented on: https://github.com/hackforla/website/issues/4590#issuecomment-1533537329 at 05/03/2023, 11:50 AM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4550#issuecomment-1533992385 at 05/03/2023, 07:23 PM PDT
- angieyan submitted review: https://github.com/hackforla/website/pull/4550#pullrequestreview-1413850974 at 05/04/2023, 01:22 PM PDT
- angieyan opened: https://github.com/hackforla/website/pull/4599 at 05/04/2023, 04:40 PM PDT
- angieyan PR was merged: https://github.com/hackforla/website/pull/4599#event-9196359814 at 05/08/2023, 09:25 PM PDT
- angieyan issue closed by PR 4599: https://github.com/hackforla/website/issues/4590#event-9196359897 at 05/08/2023, 09:25 PM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4666#issuecomment-1548835761 at 05/15/2023, 06:30 PM PDT
- angieyan submitted review: https://github.com/hackforla/website/pull/4666#pullrequestreview-1427575374 at 05/15/2023, 06:44 PM PDT
- angieyan commented on: https://github.com/hackforla/website/pull/4661#issuecomment-1548846744 at 05/15/2023, 06:47 PM PDT
- angieyan submitted review: https://github.com/hackforla/website/pull/4661#pullrequestreview-1431677207 at 05/17/2023, 02:21 PM PDT
- angieyan issue closed as completed: https://github.com/hackforla/website/issues/3813#event-11589641365 at 01/24/2024, 01:09 PM PST
- angieyan commented on: https://github.com/hackforla/website/issues/3813#issuecomment-3172936122 at 08/10/2025, 03:31 PM PDT

**Comment by angieyan** [comment]
Dear Bonnie,

I switched my career to bookkeeping/tax preparation, so please delete me
from the data.
Thank you for the opportunity to work with you.

Best regards,
Angelina
